hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCp.java

```diff
@@ -139,10 +139,9 @@ public int run(String[] argv) {
     }
     
     try {
-      context = new DistCpContext(OptionsParser.parse(argv));
-      checkSplitLargeFile();
-      setTargetPathExists();
-      LOG.info("Input Options: " + context);
+      DistCpContext ctx = new DistCpContext(OptionsParser.parse(argv));
+      LOG.info("Input Options: " + ctx);
```

Comment:
move to using {} in log...the + is only there because it was originally commons-logging